OpenEdXMobile/src/test/java/org/edx/mobile/view/WebViewActivityTest.java

```diff
@@ -18,15 +19,18 @@
 
 import static junit.framework.Assert.assertEquals;
 import static junit.framework.Assert.assertNotNull;
-import static junit.framework.Assert.assertTrue;
+import static org.assertj.android.appcompat.v7.api.Assertions.assertThat;
 
 public class WebViewActivityTest extends BaseTestCase {
 
+    // TODO: Convert to Parameterized test once we have a Robolectric parameterized test runner.
+
```

Comment:
Extra newline; the comment should be directly associated with the `test_StartWebViewActivity_LoadsUrlAndShowsTitle()` method I guess.